### DISCUSSION TITLE
What does the vertical bar between the numbers and the code mean?

### DISCUSSION THREAD

**Discussion by AdamEkelof** [helix-editor/helix#5774] [Q&A]
This is in a cpp header file, what does the black vertical line between the line numbers and the code indicate? It is not found in all files that I open. I thought it might be related to git but it remains even after commiting and pushing the change to github.
https://imgur.com/dbhtQWg

**Comment by kirawi** [comment]
Git diff lines don't yet automatically update (you have to use `:reload-all`).
